# 16 Can one block of except statements handle multiple exception?
'''
 Yes, you can use a single except block to handle multiple exceptions in Python. To do this, you can specify
 a tuple of exception types inside the except block.
 ex:- except (execetipn 1....,exception 2.....)
'''
